packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -249,6 +249,7 @@ class FlutterDevice {
     Restart? restart,
     CompileExpression? compileExpression,
     GetSkSLMethod? getSkSLMethod,
+    FlutterProject? flutterProject,
```

Comment:
does this argument ever get referenced?

Reply:
nice catch, I confused myself with all the drilling. fixed